lib/maze.rb

```diff
@@ -7,7 +7,7 @@
 # Glues the various parts of MazeRunner together that need to be accessed globally,
 # providing an alternative to the proliferation of global variables or singletons.
 module Maze
-  VERSION = '6.9.6'
+  VERSION = '6.9.7'
```

Comment:
Actually I'd go for `6.10.0`:
- major - breaking break
- minor - new feature (which this is)
- patch - bug fix (what that is exactly is open to debate sometimes)